baybe/targets/deprecation.py

```diff
@@ -6,17 +6,17 @@
 from typing import TYPE_CHECKING
 
 if TYPE_CHECKING:
-    from baybe.objective import Objective as NewObjective
+    from baybe.objectives.base import Objective as NewObjective
 
 
 def Objective(*args, **kwargs) -> NewObjective:
```

Comment:
Since we discussed the topic of deprecations today: Should this PR also remove this old deprecation? Imo, we do not need to keep this deprecation alive, and could already remove it now.

Reply:
I though we have agreed on a clear policy? 😄

Reply:
I'll close this and follow what we have agreed on in the meeting. If you disagree, please reopen